crates/bevy_ecs/src/reflect/entity_commands.rs

```diff
@@ -198,12 +202,16 @@ fn insert_reflect(
         panic!("error[B0003]: Could not insert a reflected component (of type {type_path}) for entity {entity:?} because it doesn't exist in this World. See: https://bevyengine.org/learn/errors/b0003");
     };
     let Some(type_registration) = type_registry.get(type_info.type_id()) else {
-        panic!("Could not get type registration (for component type {type_path}) because it doesn't exist in the TypeRegistry.");
+        panic!("could not get type registration (for component type {type_path}) because it doesn't exist in the TypeRegistry.");
```

Comment:
And this one I would write like this:
```suggestion
        panic!("`{type_path}` should be registered in type registry via `App::register<{type_path}>`");
```